src/Main.cc

```diff
@@ -213,8 +213,14 @@ int main(int argc, char* argv[])
 {
   // This avoids problems with locale setting dependent
   // C functions like strtod (e.g. 0.5 vs 0,5).
-  setenv("LC_ALL", "en_US.UTF-8", /* override */ 1);
-  setenv("LANG", "en_US.UTF-8", /* override */ 1);
+  const char* locales[] = {"en_US.UTF-8", "C.UTF-8", "C"};
+  for (auto locale : locales) {
+    if (std::setlocale(LC_ALL, locale) != nullptr) {
+      break;
+    }
+    // No logger yet
+    printf("Locale %s not found\n", locale);
```

Comment:
This causes all regression tests to fail for me, because the additional output, e.g. `Locale en_US.UTF-8 not found`, does not match against the test reference. Maybe it is better to not print the selected locale?

```suggestion
```

Reply:
Removed